docs/sphinx/tutorial.rst

```diff
@@ -3,6 +3,131 @@
 Tutorial
 ========
 
+.. _tutorial_units:
+
+Units of measurement
+--------------------
+
+Many of the metadata interfaces provide methods to get or set values
+with an associated unit of measurement.  The reason for this is to
+ensure that values with a unit are always associated with a unit, and
+to enforce compile-time or run-time sanity checks to ensure that the
+correct unit type is used, and that any unit conversions performed are
+legal.  The following terminology is used:
+
+dimension
+  A measured property, for example length, pressure, temperature or
+  time.
+
+unit system
+  A system of units for a given dimension, for example the SI units
+  for the dimension such as metre, pascal, celsius or second; multiple
+  systems may be provided for a given dimension, such as Imperial
+  length units, bar or Torr for pressure or fahrenheit for
+  temperature.
+
+unit
+  A unit of measure within a given unit system, for example cm, µm, nm
+  are all scaled units derived from m.
+
+base unit
+  The primary unit for a given unit system; all other units are scaled
+  relative to this unit.  Automatic conversion between unit systems is
+  defined in conversion of base units.
+
+quantity
+  A measured value with an associated unit.
+
+.. _tutorial_basic_units:
+
+Basic units
+^^^^^^^^^^^
+
+Basic units are provided by a static compile-time type-safe unit
+system based upon Boost.Units.  All the data types provided are simply
+typedefs or specialisations of the Boost.Units library unit types.
+
+As a user of the library, quantity types are the primary type which
+you will use.  The other types are implementation details which will
+not be needed unless you have a need to add additional custom units to
+the existing set of registered units.  However, a comprehensive set of
+SI, Imperial and other customary units are provided.  A full list of
+quantity types is listed in the :ome_common_api:`units namespace
+<namespaceome_1_1common_1_1units.html>`.
+
+To create a quantity, use the :cpp:func:`quantity::from_value` method,
+which will return a :cpp:class:`quantity` of the desired type.  Since
+the unit is encoded in the type name, the unit type is immutable and
+correctness is enforced at compile time.  This quantity is freely
+assignable to any quantity of the same type, and arithmetic operations
```

Comment:
Would 'any quantity of the same unit system' make sense here? Unit system is defined in the terminology but not used otherwise and might be a bit clearer